src/components/Textfield/styles.scss

```diff
@@ -1,4 +1,6 @@
-@import '../../../style-dictionary-dist/textfield_variables.scss';
+@use 'sass:math';
```

Comment:
I see we are using math in the button component for example and not having this @use  what is it for?

Reply:
This was just copied over.
I believe sass:math is needed fi we are using math otherwise you get warnings.
Cleaning up css is not the scope of this PR.